app/pages/sections/routes.js

```diff
@@ -195,6 +196,12 @@ export const sectionRoutes = (authProvider, addContext, sessionManager) => {
     const { orderId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     await getFundingSource({ orderId, accessToken });
+    getOrderDescription({
+      req,
+      sessionManager,
+      accessToken,
+      logger,
+    });
```

Comment:
Is there a reason why the results of `getFundingSource` and `getOrderDescription` aren't used?

`getOrderDescription` is async so will need an `await` too.

Reply:
yeah the story is just for calling it
will use them in the next story for create basic page